core/src/main/java/org/springframework/security/core/userdetails/UserDetails.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2004, 2005, 2006 Acegi Technology Pty Limited
+ * Copyright 2004-2023 Acegi Technology Pty Limited
```

Comment:
@ahmd-nabil the Acegi copyright headers are a special case. Will you please leave those as they are?

Reply:
sure

Reply:
@jzheaux done, thanks for your great support btw. I am waiting for your review on PR #14265 whenever you have time.